assets/src/blocks/amp-story-post-author/index.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { getMetaBlockSettings } from '../../helpers';
+import './edit.css';
+
+export const name = 'amp/amp-story-post-author';
+
+export const settings = {
+	title: __( 'Story Author', 'amp' ),
+	description: __( 'Displays the name of the story\'s author', 'amp' ),
```

Comment:
Should this not use a non-straight quote like `story‘s`?

Reply:
I don't think we use these kind of quotes anywhere in the plugin nor in WordPress core. But I might use double quotes so I can get rid of the backslash.

Reply:
I did a search. They're being used in Gutenberg.

Reply:
For example: https://github.com/WordPress/gutenberg/blob/0b93ad47797a1ba2cea3c83a7f973d33abcd351f/packages/editor/src/components/post-locked-modal/index.js#L171

Reply:
I've always been told to change straight quotes to curly quotes when I've committed such strings to core.